<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>427</width>
    <height>640</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OneFM</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image:url(&quot;:/images/background.png&quot;);</string>
   </property>
   <widget class="QToolButton" name="buttonMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>37</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>37</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>37</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border: none;
margin: 0px;
padding: 0px;
background-image: url(:/images/menu.png)
}

QToolButton:hover{

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>96</width>
      <height>96</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border: none;
padding: 0px;
margin: 0px;
background-image:url(&quot;:/images/logo1.png&quot;);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonVolumeUp">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>138</y>
      <width>28</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>28</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border: none;
margin: 0px;
padding: 0px;
background-image:url(&quot;:/images/up.png&quot;);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonVolumeDown">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>492</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>28</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border: none;
margin: 0px;
padding: 0px;
background-image:url(&quot;:/images/down.png&quot;);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderVolume">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>62</width>
      <height>312</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>62</width>
      <height>312</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>62</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
background-image:url(&quot;:/images/slide.png&quot;);
background-repeat:no-repeat;
}

QSlider::groove:vertical{
border: none;
padding:0px;
margin:0px;
}

QSlider::sub-page:vertical {
border: none;
padding:0px;
margin:0px;
}

QSlider::add-page:vertical {
border: none;
padding:0px;
margin:0px;
}

QSlider::handle:vertical{
background-image:url(&quot;:/images/volume.png&quot;);
width: 59px;
height: 39px;
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelCenter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>200</width>
      <height>629</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/images/center0.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonRefresh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>37</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/images/refresh.png);
border:none;
padding:0px;
margin:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonPlay">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>475</y>
      <width>130</width>
      <height>129</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border: none;
padding:0px;
margin:0px;
background-image:url(:/images/play.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelHand">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>35</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border:none;
padding:0px;
margin:0px;
background-image:url(:/images/hand.png)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrentArtist">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border:none;
background:none;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Apasa butonul play</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonStar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>35</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border:none;
margin:0px;
padding:0px;
background-image:url(:/images/star.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTopLogo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding:0px;
border:0px;
background:none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
